client/tests/integration/upgrade.rs

```diff
@@ -241,25 +240,10 @@ fn executor_custom_instructions_complex() -> Result<()> {
     use executor_custom_data_model::complex::{
         ConditionalExpr, CoreExpr, CustomInstructionExpr, EvaluatesTo, Expression, Greater,
     };
-    use iroha_config::parameters::actual::Root as Config;
 
-    let mut config = Config::test();
-    // Note that this value will be overwritten by genesis block with NewParameter ISI
-    // But it will be needed after NewParameter removal in #4597
-    config.chain_wide.executor_runtime.fuel_limit = 1_000_000_000;
-
-    let (_rt, _peer, client) = PeerBuilder::new()
-        .with_port(11_275)
-        .with_config(config)
-        .start_with_runtime();
+    let (_rt, _peer, client) = PeerBuilder::new().with_port(11_275).start_with_runtime();
     wait_for_genesis_committed(&vec![client.clone()], 0);
 
-    // Remove this after #4597 - config value will be used (see above)
-    let parameters = ParametersBuilder::new()
-        .add_parameter(EXECUTOR_FUEL_LIMIT, Numeric::from(1_000_000_000_u32))?
-        .into_set_parameters();
-    client.submit_all_blocking(parameters)?;
```

Comment:
I think it is needed to set EXECUTOR_FUEL_LIMIT with SetParameter ISI, otherwise this test will fail